src/Components/Locations.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export default class Locations extends Component {
+  constructor() {
+    super();
+    this.state = {
+      locations: [],
+      showList: false,
+    };
+  }
+
+  getLocations = async () => {
+    const { data } = await axios.get("https://ghibliapi.herokuapp.com/locations");
+
+    this.setState({ 
```

Comment:
This spacing/whitespace use is a little odd. I'd highly recommend installing the Prettier vscode extension and setting it up to run automatically when you save a file.